CONTRIBUTING.md

```diff
@@ -59,9 +68,22 @@ workflow.
 5. Set line length to 120
 6. Install mypy plugin
 
+## Testnets and review environments
+
+With the launch of `1.0.0` we will begin running an official `testnet`.  
+Prior to the release of `1.1.0` there will be two running test nets. `1.0` and `1.1b`. The `1.1b` testnet will be a beta of the pending 1.1 release, which will enable transactions on the chia blockchain.
```

Comment:
When I read this, I ask myself "Why does the `1.1b` have a `b` suffix and `1.0.0` doesn't?"

Reply:
Note that naming a testnet for a software release doesn't seem like a good practice long-term. Presumably, all networks will work with all versions of the software as long as the parameterized constants (including genesis challenge) match. The sole exception is 1.0.x, which has a shelf life and fails after `INITIAL_FREEZE_PERIOD`. We might say "this release runs on testnet X by default" but we really ought to work toward a work where the software knows about all named testnets and mainnet(s) and their constants, and MAYBE has a different default of which one to select (or maybe the GUI chooses a different default), although any known nets can be selected from the command-line.

Reply:
TLDR: I don't think we should call the testnets `1.0` and `1.1`.

Reply:
I agree all testnets should have the same constants after 1.1. But before that date, they are not compatible right? I'm not opposed to using different names though.

Reply:
The key difference to me seems to be that one of them is intended to be used primarily to test the transition from 1.0.0 to 1.1.0 (ie. removing the "die after `INITIAL_FREEZE_PERIOD`). But then 1.1.0 will work with both testnets. The main difference seems to be the constants chosen by default. Both testnets will be supported by both 1.0.0 and 1.1.0 (except 1.0.0 will die very soon on one of the testnets since we want transactions right away).

Reply:
after the release of 1.1.0 there will only be one testnet. it will be called "testnet" 

These are ecosystems entirely for our benefit, and end users can join them if they wish. 

The goal of the testnet is to run a known version of software that we can test changes against with as close to live interaction as possible. In these cases were testing possible breaking changes for a new release, and the post transaction drop. were essentially running two parallel versions of what will eventually be just one testnet. barring a potential breaking change in a later release we feel necessitates its own testing ecosystem. 

These are named for the software version being tested because that is the most logical way in my mind to keep track of these many test cases. I am open to naming them to something more descriptive!

Perhaps we can go with "testnet" and "transaction-beta-testnet"?

Reply:
Yeah, even if we were hard-forking every month, it would probably be better to name the testnets in a different namespace than the software.

JDE also needs a phrase that means "what version of the software are our machines in the colo running?" And in Sysadmin parlance, the default way to say that is "testnet is running on 1.x.y". But that is obviously not correct for this project, because the testnet is a swarm of different versions.

So, if anyone thinks of a good way to say that, let JDE know :)